docs/integration.rst

```diff
@@ -605,15 +605,109 @@ These integrations allow you to perform various operations within various servic
      - Operators
      - Sensors
 
+   * - `Atlassian Jira <https://www.atlassian.com/pl/software/jira>`__
+     -
+     - :mod:`airflow.contrib.hooks.jira_hook`
+     - :mod:`airflow.contrib.operators.jira_operator`
+     - :mod:`airflow.contrib.sensors.jira_sensor`
+
+   * - `Databricks <https://databricks.com/>`__
+     -
+     - :mod:`airflow.contrib.hooks.databricks_hook`
+     - :mod:`airflow.contrib.operators.databricks_operator`
+     -
+
+   * - `Datadog <https://www.datadoghq.com/>`__
+     -
+     - :mod:`airflow.contrib.hooks.datadog_hook`
+     -
```

Comment:
```suggestion
```

Do we need this space?

Reply:
We need to create a list with a list, so indentations are needed.